Makefile

```diff
@@ -10,9 +10,9 @@ IMAGE = "flaconi/atlantis-terragrunt"
 TAG = latest
 
 # Versions
-ATLANTIS = '0.17.5'
-TERRAFORM = '1.1.1'
-TERRAGRUNT = '0.35.14'
+ATLANTIS = '0.17.6'
+TERRAFORM = '1.1.2'
```

Comment:
Talking about quick iterations... We just got on `1.1.1` after updating to `1.1.0`. We might benefit from letting the dust settle a bit. I know they're just patch version, though...

Reply:
I do not plan to update instantly to `1.1.2`. atlantis container `0.17.6` does not have TF `1.0.x` preinstalled by whatever reason. But I created already a container with it anyway.